Formula/ansible.rb

```diff
@@ -588,14 +586,14 @@ def install
     EOS
     (testpath/"hosts.ini").write [
       "localhost ansible_connection=local",
-      " ansible_python_interpreter=#{Formula["python@3.10"].opt_bin}/python3",
+      " ansible_python_interpreter=#{Formula["python@3.10"].opt_bin}/python3.10",
       "\n",
     ].join
     system bin/"ansible-playbook", testpath/"playbook.yml", "-i", testpath/"hosts.ini"
 
     # Ensure requests[security] is activated
     script = "import requests as r; r.get('https://mozilla-modern.badssl.com')"
-    system libexec/"bin/python3", "-c", script
+    system libexec/"bin/python", "-c", script
```

Comment:
This isn't strictly needed, but it doesn't hurt, and it'll prevent this from showing up in the regex I used at #108008.